src/buildMathML.js

```diff
@@ -471,6 +493,65 @@ groupTypes.underline = function(group, options) {
     return node;
 };
 
+groupTypes.accentunder = function(group, options) {
+    const accentNode = stretchy.mathMLnode(group.value.label);
+    const node = new mathMLTree.MathNode(
+        "munder",
+        [buildGroup(group.value.body, options), accentNode]
+    );
+    node.setAttribute("accentunder", "true");
+    return node;
+};
+
+groupTypes.strikeThru = function(group, options) {
+    const node = new mathMLTree.MathNode(
+        "memclose", [buildGroup(group.value.body, options)]);
+    let notation = "";
+    switch (group.value.label) {
+        case "\\bcancel":
+            notation = "downdiagonalstrike";
+            break;
+        case "\\sout":
+            notation = "horizontalstrike";
+            break;
+        default:
+            notation = "updiagonalstrike";
+    }
+    node.setAttribute("notation", notation);
+    return node;
+};
+
+groupTypes.boxed = function(group, options) {
+    const inner = buildExpression(group.value.value, options);
+    const node = new mathMLTree.MathNode("menclose", inner);
+    node.setAttribute("notation", "box");
+    return node;
+};
+
+groupTypes.horizBrace = function(group, options) {
+    const accentNode = stretchy.mathMLnode(group.value.label);
+    return new mathMLTree.MathNode(
+        (group.value.isOver ? "mover" : "munder"),
+        [buildGroup(group.value.base, options), accentNode]
+    );
+};
+
+groupTypes.xArrow = function(group, options) {
+    const arrowNode = stretchy.mathMLnode(group.value.label);
+    const upperNode = buildGroup(group.value.body, options);
+    let node;
+
+    if (group.value.below) {
+        const lowerNode = buildGroup(group.value.below, options);
+        node = new mathMLTree.MathNode(
+            "munderover", [arrowNode, lowerNode, upperNode]
+        );
```

Comment:
Give `\xleftarrow[123]{}` would `upperNode` be empty?  In that case maybe we should use `munder`?